docs/sdk/v3/guides/liquidity/01-minting-a-position.md

```diff
@@ -3,4 +3,90 @@ id: minting
 title: Minting a Position
 ---
 
-TODO
\ No newline at end of file
+## Introduction
+
+This guide will cover how to create (or mint) a liquidity position on the Uniswap V3 protocol.
+It is based on the [Minting a position code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/minting-position), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/quoting/README.md) and follow the setup instructions.
+
+:::info
+
+For a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background coverage](./01-background.md) page!
+
+:::
+
+In this example we will use  the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair.
+The inputs are the **token in**, the **token out**,  the **amount in**, the **fee** and the **liquidity**.
```

Comment:
Can we describe what these represent better vs their in code names?

1. What is the units/type of the liquidity parameter?
2. What does the fee refer to?

Even on the token examples, saying "input token" and "output token" makes more sense, though that's a nit. The other two we should explain better!